library/std/src/panic.rs

```diff
@@ -461,5 +461,41 @@ pub fn resume_unwind(payload: Box<dyn Any + Send>) -> ! {
     panicking::rust_panic_without_hook(payload)
 }
 
+/// Make all future panics abort rather than unwinding.
```

Comment:
This might make it sound like this only has effect in `panic = "unwind"` situations. But even in `panic = "abort"` this still prevents the panic hook from running.

Maybe something like this:
```suggestion
/// Make all future panics abort directly without running the panic hook or unwinding.
```